### PR TITLE
Fixed Designer page problems

### LINKED ISSUE #15720: Designer adding all available tables to a designer page after adding a new relationship.
###  Problem (Large): 
    Designer adding all available tables to a designer page after adding a new relationship.
  
###  Steps to replicate:
  
    a.  build a new ER diagram and save it with a new name,
    b.  reload the diagram,
    c.  select "Show/Hide tables list" from (default) left-side menu,
    d.  click any checked table name from list of tables shown in right-side list to remove it.
    e.  NOTE that the "modified" indicator (an asterisk) is not displayed in Diagram Name (expected),
        nor is `table_coords updated (not expected).
    f.  save the page - `table_coords IS updated (expected).
    g.  reload the page - requested table in step d. is removed.
    h.  select "Create Relationship" from (default) left-side menu and build a new relationship.
    i.  NOTE that the "modified" indicator (an asterisk) is not displayed in Diagram Name (expected),
        nor is `table_coords updated (not expected).
    j.  save the page - `table_coords` IS updated (expected) but ALL available tables are now 
        associated with the currentpage.
    k.  The screen display is NOT updated, however, but ALL tables are checked after selecting 
        "Show/Hide tables list".
    l.  select "Reload" from (default) left-side menu displays the current page with ALL tables
       displayed.
    
###  Expected result:
  
    During step i. above, the "modified" indicator should be displayed.
    During step j. above, `table_coords` should NOT be updated unless a table has been added or
    removed from the display.
    
    
  
###  Problem (Small): 
    Designer not changing the tooltip 'title' of the checkbox selecting tables to show or hide
    from list of tables shown when selecting "Show/Hide tables list" from (default) left-side menu.
    
###  Steps to replicate:
  
    a.  display any designer page.
    b.  select "Show/Hide tables list" from (default) left-side menu,
    c.  hover the mouse cursor over any checked table show/hide checkbox.  Tooltip "Hide" is 
        displayed.
    d.  click any of the checked table checkboxes to remove it from the displayed page.
    e.  hover the mouse cursor over checkbox clicked in step d (now NOT checked).  Tooltip "Hide" is
         STILL displayed.
    
###  Expected result:
  
    The tooltip shown when hovering over the "Show/Hide table name" checkbox should reflect the 
    action that  will occur if the checkbox is clicked to change state.

### Server configuration
- Apache/2.4.41 (Win64) OpenSSL/1.1.1c PHP/7.3.10

- Windows 10
- Apache/2.4.41 (Win64)
- OpenSSL/1.1.1c 
- Database version: libmysql - mysqlnd 5.0.12-dev (MariaDB)
- PHP version: PHP/7.3.10
- phpMyAdmin version: 5.0.0 (also in 4.9.2 - likely 4.9.3)

### Client configuration

- Browser:  79.0.3945.88 (Official Build) (64-bit) (cohort: Stable)
- Operating system: Windows 10 OS Version 1903 (Build 18362.535)

### Additional context
JS V8 7.9.317.32

### PR DESCRIPTION
**Pull request by yashrajbothra**
### Description

1. made a call to markUnsaved function to set asterisk.
2. set value of checkbox only when table is displayed.
3. change tooltip message when clicked

Fixes #15720 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

**Comment by yashrajbothra** [comment]
Hey @williamdes  kindly review the PR

File: js/designer/move.js [hunk 1]
```diff
@@ -560,6 +561,7 @@ DesignerMove.addTableToTablesList = function (index, tableDom) {
         DesignerMove.selectTab($(this).attr('designer_url_table_name'));
     });
     $($newTableLine).find('.scroll_tab_checkbox').click(function () {
+        $(this).attr('title',(i,currentvalue) => { return currentvalue == 'Show' ? 'Hide' : 'Show' });
```
Comment (williamdes) [inline js/designer/move.js]: That will not work using another language than English

**Review by williamdes — CHANGES_REQUESTED** [review]

File: js/designer/move.js (hunk 1 above)
Comment (yashrajbothra) [reply js/designer/move.js]: Changed messages to msg variables. Kindly review the PR

File: js/designer/move.js [hunk 2]
```diff
@@ -560,6 +561,7 @@ DesignerMove.addTableToTablesList = function (index, tableDom) {
         DesignerMove.selectTab($(this).attr('designer_url_table_name'));
     });
     $($newTableLine).find('.scroll_tab_checkbox').click(function () {
+        $(this).attr('title',(i,currentvalue) => { return currentvalue == Messages.strHide ? Messages.strShow : Messages.strHide });
```
Comment (williamdes) [inline js/designer/move.js]: Oh, 2 tings

`=>` https://caniuse.com/#feat=arrow-functions please use a normal function

`==` => `===` can be replaced my a strict equals

**Review by williamdes — APPROVED** [review]

File: js/designer/move.js [hunk 3]
```diff
@@ -529,6 +529,8 @@ DesignerMove.addTableToTablesList = function (index, tableDom) {
     var table = $(tableDom).find('.small_tab_pref').attr('table_name');
     var dbEncoded = $(tableDom).find('.small_tab_pref').attr('db_url');
     var tableEncoded = $(tableDom).find('.small_tab_pref').attr('table_name_url');
+    var tableIsChecked = $(tableDom).css('display') == "block" ? "checked" : "";
```
Comment (williamdes) [inline js/designer/move.js]: a strict equals can be used here also

File: js/designer/move.js (hunk 2 above)
Comment (yashrajbothra) [reply js/designer/move.js]: Thanks,I will keep these things in mind.

File: js/designer/move.js (hunk 3 above)
Comment (yashrajbothra) [reply js/designer/move.js]: yea sure

Comment (yashrajbothra) [reply js/designer/move.js]: Do i have to commit and push these changes?

Comment (williamdes) [reply js/designer/move.js]: Yes

Comment (yashrajbothra) [reply js/designer/move.js]: Done!

Comment (williamdes) [reply js/designer/move.js]: Thank you

**Review by williamdes — APPROVED** [review]

**Comment by williamdes** [comment]
Waiting for the CI and will test your work manually

**Comment by NKarasek** [comment]
@yashrajbothra @williamdes 
Good job folks and thank you for being so responsive.  I was just about to note that the checkboxes were not being updated with their new title when clicked - but @williamdes did that one line change as I looked at the list of comments.  Again - thanks much - and Happy New Year to you both!!

**Comment by yashrajbothra** [comment]
Thanks @NKarasek @williamdes and a very Happy New year.

**Comment by williamdes** [comment]
@Tithugues can you please review this PR ?
You introduced this in #15448 de57539fb71beacba54b18283893ca3cba6f01cf

@yashrajbothra can you check it does not revert #15446 ?

**Comment by Tithugues** [comment]
It sounds good to me.

**Comment by yashrajbothra** [comment]
> @yashrajbothra can you check it does not revert #15446 ?

I checked #15446 and as I have only added some changes it will not revert the PR.

**Comment by williamdes** [comment]
Thank you @yashrajbothra for your contribution !

**Pull request merged**